web/src/user_group_create.ts

```diff
@@ -255,6 +255,9 @@ export function set_up_handlers(): void {
     user_group_create_members.create_handlers($people_to_add_holder);
 
     const $container = $("#user-group-creation").expectOne();
+    $(window).on("resize", () => {
```

Comment:
Instead of having these handlers, I guess we can call them in `resize.resize_page_components`. For avoiding unnecessary resizing, the functions can be modified to return early if the overlay is not opened.